src/datepicker/__tests__/datepicker.test.js

```diff
@@ -260,6 +260,21 @@ describe('Datepicker', () => {
     expect(onChange.mock.calls).toHaveLength(1);
   });
 
+  // Issue #3230
+  test('does not fire the onchange when date length is less than default formatDate with single date when no formatString is passed', () => {
+    const onChange = jest.fn();
+    const component = mount(<Datepicker onChange={onChange} value={null} />);
+
+    // $FlowFixMe
```

Comment:
I don't have flow extension installed so I guess if there's an error in above test it also would be in this one so I kept $FlowFixMe as it is